packit_service/worker/helpers/testing_farm.py

```diff
@@ -1298,6 +1298,22 @@ def test_target2build_target(self, test_target: str) -> str:
         """
         return self.test_target2build_target_for_test_job(test_target, self.job_config)
 
+    def is_manual_job_being_triggered(self) -> bool:
+        """
+        Is this job meant to be triggered manually
+        and is it being triggered by a matching
+          /packit test --labels label
+        command?
+        """
+        if self.job_config.manual_trigger:
+            if self.comment_arguments.labels:
+                return (
+                    True
+                    if self.job_config.identifier in self.comment_arguments.labels
+                    else False
+                )
+        return False
```

Comment:
```suggestion
        if self.job_config.manual_trigger and self.comment_arguments.labels:
                return self.job_config.identifier in self.comment_arguments.labels
        return False
```

Reply:
I think we should check here both identifier and labels? Something like we have in pre-checks [here](https://github.com/packit/packit-service/blob/9bbb1d743d02166a97070c4bfd9e7cf647c1eefe/packit_service/worker/checker/testing_farm.py#L165) and [here](https://github.com/packit/packit-service/blob/9bbb1d743d02166a97070c4bfd9e7cf647c1eefe/packit_service/worker/checker/testing_farm.py#L132)?